src/main/java/core/basesyntax/service/impl/SupplyAction.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ActionHandler;
+
+public class SupplyAction implements ActionHandler {
+    public void apply(String fruit, int quantity) {
```

Comment:
The `apply` method in the `ActionHandler` interface should return a boolean indicating the success of the operation, but here it is returning void. Please make sure the method signature is consistent with the interface.